docs/source/json-api/index.rst

```diff
@@ -251,6 +253,105 @@ Nonempty Response with Unknown Template IDs Warning
         "status": 200
     }
 
+WebSocket ``/contracts/searchForever``
+======================================
+
+List currently active contracts that match a given query, with
+continuous updates.
+
+Two subprotocols must be passed, as described in `Choosing a party
+<#choosing-a-party>`__.
+
+application/json body must be sent first, formatted according to the
+:doc:`search-query-language`::
+
+    {"%templates": ["Iou:Iou"]}
+
+output a series of JSON documents, each ``argument`` formatted according
+to :doc:`lf-value-specification`::
+
+    {
+        "created": [{
+            "observers": [],
+            "agreementText": "",
+            "signatories": ["Alice"],
+            "contractId": "#1:0",
+            "templateId": "398e67533888ab6532c3e62c2c3445182e2cdd291457732f174d2698a7d1db2d:Iou:Iou",
+            "witnessParties": ["Alice"],
```

Comment:
I remember removing `witnessParties` in favor of `signatories` and `observers`.

Reply:
Replaced in 42dc6b964d58ed8555c3240c816715f024c263fe; thanks.